default_gateway.go

```diff
@@ -103,6 +103,9 @@ func (sb *sandbox) needDefaultGW() bool {
 		if ep.getNetwork().Type() == "null" || ep.getNetwork().Type() == "host" {
 			continue
 		}
+		if ep.joinInfo.disableGatewayService {
+			return false
+		}
```

Comment:
When a driver sets `disableGatewayService`, i assume that the driver is capable of providing the default route for that container. Hence even if 1 endpoint suggests it can provide the default route for the sandbox, then this function must return false as it is implemented now.
Hence, the current change of returning false looks appropriate.